lesson02/index.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="style.css" type="text/css" />
+    <title>TerraceTechフロントエンドエンジニア養成所</title>
+  </head>
+  <body>
+    <div>
+      <ul id="js-lists"></ul>
```

Comment:
IMO
I learned it from Morita-san too, and I think `ul` is appropriate here.
Because `ul` is a list and `li` is a listitem.
If you use lists, it will look as if there are several `ul`.
[リスト系のコンポーネント名 Images と ImageList はどっちが有利か](https://qiita.com/misuken/items/d62c304fd6d730801ddb)

Reply:
I'll be careful with the plural.
Thanks for letting me know!